src/schema/erc20.graphql

```diff
@@ -35,3 +35,21 @@ type ERC20Approve implements DecodedTransaction {
   value: String
   tokenContract: TokenContract
 }
+
+type ERC20TransferEvent implements DecodedLog {
+  entity: Entity
+  standard: String
+  event: String
```

Comment:
What do the fields 'entity', 'standard', and 'event' represent here?

Reply:
Good catch. I'm enhancing the docs in the schema.